CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ __BEGIN_UNRELEASED__
 ### Security
 __END_UNRELEASED__
 
+## [0.7.0] - 2019-08-14
```

Comment:
`0.7.0` -> `0.7.1` and `2019-08-14` -> `2019-08-15`

Reply:
Ah, nice catch...